examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Api.Gax;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    using Lib;
+    using Google.Ads.GoogleAds.V3.Errors;
+    using Google.Ads.GoogleAds.V3.Services;
+    using System;
+
+    /// <summary>
+    /// This example gets the account hierarchy of the specified manager account. If you don't specify
+    /// manager customer ID, the example will instead print the hierarchies of all accessible customer
+    /// accounts for your authenticated Google account.
+    /// Note that if the list of accessible customers for your authenticated Google account includes
+    /// accounts within the same hierarchy, this example will retrieve and print the overlapping
+    /// portions of the hierarchy for each accessible customer.
+    /// </summary>
+    public class GetAccountHierarchy : ExampleBase
+    {
+        // private const string MANAGER_CUSTOMER_ID = "3955750366";//"INSERT_MANAGER_CUSTOMER_ID_HERE";
+        private const int PAGE_SIZE = 1000;
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            GetAccountHierarchy codeExample = new GetAccountHierarchy();
+            Console.WriteLine(codeExample.Description);
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+            
+            try
+            {
+                codeExample.Run(new GoogleAdsClient(), customerId);
+            }
+            catch (GoogleAdsException adsException)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {adsException.Message}");
+                Console.WriteLine($"Failure: {adsException.Failure}");
+                Console.WriteLine($"Request ID: {adsException.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description =>
```

Comment:
This looks different from our other code examples like https://github.com/googleads/google-ads-dotnet/blob/master/examples/BasicOperations/GetExpandedTextAds.cs. Shouldn't this be a 'get' rather than an override?

Reply:
Reverted expression body syntax to be consistent with other exampels.